docs/output.md

```diff
@@ -2,58 +2,265 @@
+
+### VEP
+
+[VEP (Variant Effect Predictor)](https://www.ensembl.org/info/docs/tools/vep/index.html), based on `Ensembl`, is a tool to determine the effects of the variants. The generated `VCF` header contains the software version, also the version numbers for additional databases like `Clinvar` or `dbSNP` used in the `VEP` line.
+The format of the [consequence annotations](https://www.ensembl.org/info/genome/variation/prediction/predicted_data.html) is also in the `VCF` header describing the `INFO` field.
+
+Currently, it contains:
+
+- `Consequence`: impact of the variation, if there is any
+- `Codons`: the codon change, i.e. cGt/cAt
+- `Amino_acids`: change in amino acids, i.e. R/H if there is any
+- `Gene`: ENSEMBL gene name
+- `SYMBOL`: gene symbol
+- `Feature`: actual transcript name
+- `EXON`: affected exon
+- `PolyPhen`: prediction based on [PolyPhen](http://genetics.bwh.harvard.edu/pph2/)
+- `SIFT`: prediction by [SIFT](http://sift.bii.a-star.edu.sg/)
+- `Protein_position`: Relative position of amino acid in protein
+- `BIOTYPE`: Biotype of transcript or regulatory feature
+
+To annotate variants using `vep`, you can use `--annotate_tools vep`.
+The annotated variant files in VCF format can be found in `results/variant_annotation` folder.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `results/variant_annotation/[SAMPLE]/`
+  - `[SAMPLE]_VEP.ann.vcf.gz`: Annotated VCF from VEP process.
+  - `[SAMPLE]_VEP.ann.vcf.gz.tbi`: This is the index of the above VCF file.
+
+</details>
+
+When `--annotate_tools merge` option is used, the annotation from both `snpeff` and `vep` are combined into a single VCF file which can be found with the following naming convention.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `results/variant_annotation/[SAMPLE]/`
+  - `[SAMPLE]_snpEff_VEP.ann.vcf.gz`: Combined annotation from both snpEff and VEP.
+  - `[SAMPLE]_snpEff_VEP.ann.vcf.gz.tbi`: This is the index of the above VCF file.
+
+</details>
+
+For further reading and documentation see the [VEP manual](https://www.ensembl.org/info/docs/tools/vep/index.html)
+
+### MultiQC report with FASTQC read summary
+
+[MultiQC](http://multiqc.info) is a visualization tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in the report data directory.
+
+It integrates [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) which gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences. For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+
```

Comment:
Does it also contain other reports such as samtools stats, BQSR etc. or "just" FastQC?

Reply:
Currently only FASTQC. May be in the next (patch release) I plan to add an extra step of variant QC and then bring all of the stats (including samtools) to the multiQC report.

Reply:
Added MultiQC integration with QC stats from different tools including VEP and snpEff